File: C++/A1/059_Grader Bug.cpp
#include <iostream>
#include <stdio.h>
using namespace std;
int main(){
    string from,to;
    bool check=true;
    int A,B;
    float C;
    cin >> from >> to >> C;
    string route = from + "-" + to;
    if(route == "BKK-CNX"){A=10;B=30;}
    else if(route == "CNX-UBP"){A=15;B=40;}
    else if(route == "UBP-BKK"){A=20;B=40;}
    else if(route == "BKK-PKT"){A=25;B=50;}
    else if(route == "PKT-CNX"){A=30;B=60;}
    else if(route == "UBP-PKT"){A=40;B=70;}
    else check = false;
    if(check)printf("%.2f",A+(B*C));
    else if(!check)cout << "Error";
}       

File: Python/A1/059_Grader Bug.py
p = {
    "BKK-CNX":10,
    "CNX-UBP":15,
    "UBP-BKK":20,
    "BKK-PKT":25,
    "PKT-CNX":30,
    "UBP-PKT":40
}
w = {
    "BKK-CNX":30,
    "CNX-UBP":40,
    "UBP-BKK":40,
    "BKK-PKT":50,
    "PKT-CNX":60,
    "UBP-PKT":70
}
result = ""
a,b = map(str,input().split())
result += a+"-"+b
c = float(input())
if(result in p):
    print(f"{(p[result]+(c*w[result])):.2f}")
else:print("Error")